lib/DistributionsCore/src/DistributionsCore.jl

```diff
@@ -0,0 +1,521 @@
+module DistributionsCore
+import Base: maximum, minimum, extrema, ==
```

Comment:
Maybe we move away from using `import` and instead qualify functions explicitly when we extend them? At least I prefer that over `import`.